web3.js/test/bpf-loader.test.ts

```diff
@@ -147,6 +147,30 @@ if (process.env.TEST_LIVE) {
         );
       });
 
+      it('simulate transaction with returnData', async () => {
+        const simulatedTransaction = new Transaction().add({
+          keys: [
+            {pubkey: payerAccount.publicKey, isSigner: true, isWritable: true},
+          ],
+          programId: program.publicKey,
+        });
+        const {err, returnData} = (
+          await connection.simulateTransaction(simulatedTransaction, [
+            payerAccount,
+          ])
+        ).value;
+        const expected_return_data = new Uint8Array([1, 2, 3]);
```

Comment:
nit: camelCase in JS, snake_case in Rust
```suggestion
        const expectedReturnData = new Uint8Array([1, 2, 3]);
```